news/2020-02-19-new-featured-artist-elfensjon.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: ELFENSJóN"
+date: 2020-02-19 06:00:00 +0000
+---
+
+**ELFENSJóN** throws wide the gates to dark fantasy and Valhalla as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/69/header.jpg)
+
+Referred to us by the wonderful people of *Undead Corporation*, **ELFENSJóN** is a Japanese production group featuring a wicked fusion of dark fantasy and rock themes, all underpinned with a variety of vocal excellence.
+
+We've pulled a choice selection from their two major albums, *ASH OF ROUGE* (both the *Elisia* and *Astral* sides) and *EINHERJAR*, a name that might sound slightly familiar to anyone who's been keeping tabs on a certain minty confectionary as of late.
```

Comment:
is "pulled a choice selection" a thing? or did u mean "pulled a selection"
```suggestion
We've pulled a selection from their two major albums, *ASH OF ROUGE* (both the *Elisia* and *Astral* sides) and *EINHERJAR*, a name that might sound slightly familiar to anyone who's been keeping tabs on a certain minty confectionary as of late.
```

Reply:
it's a colloquialism which was apparently lost in translation

"choice" is like "prime" or "special" in this instance